jest.config.ts

```diff
@@ -2,8 +2,8 @@
  * For a detailed explanation regarding each configuration property and type check, visit:
```

Comment:
Why did this file get formatted just now? :eyes:  Have you updated linting config?